src/main/java/org/example/Ships/HibernateFactory.java

```diff
@@ -0,0 +1,31 @@
+package org.example.Ships;
+
+import org.example.learnig.Car;
+import org.example.learnig.Owner;
+import org.example.learnig.Wheel;
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistry;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateFactory {
```

Comment:
mozna juz wywalic to factory i skorzystac z tego w glownym katalogu aplikacji :)